includes/api/class-wc-admin-rest-onboarding-profile-controller.php

```diff
@@ -0,0 +1,330 @@
+<?php
+/**
+ * REST API Onboarding Profile Controller
+ *
+ * Handles requests to /onboarding/profile
+ *
+ * @package WooCommerce Admin/API
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Onboarding Profile controller.
+ *
+ * @package WooCommerce Admin/API
+ * @extends WC_REST_Data_Controller
+ */
+class WC_Admin_REST_Onboarding_Profile_Controller extends WC_REST_Data_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v4';
```

Comment:
There was some discussion in slack. Could we update this to `wc-internal/v1`?  p1557756996005400-slack-wc-onboarding.

Reply:
Namespaced this and went ahead and namespaced the onboarding levels controller as well.